src/skins/vector/css/matrix-react-sdk/structures/_GroupView.scss

```diff
@@ -304,6 +304,18 @@ limitations under the License.
 }
 
 .mx_GroupView_groupDesc textarea {
-    width: 50%;
+    width: 100%;
+    max-width: 100%;
     height: 150px;
 }
+
+.mx_GroupView_groupDesc_placeholder {
+    background-color: $group-my-groups-placeholder-bg;
+    color: $group-my-groups-placeholder-fg;
```

Comment:
i'm surprised this needs a custom colour? shouldn't one of the existing generic theme colours fit?

Reply:
There _might_ have been one that already existed but I guess its name felt to specific for me to reuse it somewhere else... although I have contradicted myself here by using a `$group-my-groups-*` variable in GroupView.

Reply:
e.g. the "preview-bg-color" or whatever it's called.  if it feels icky to reuse the same constant for this component, one could always rename it as "info-plinth-bg-color" or something

Reply:
`info-plinth-bg-color` +1 !